openstudiocore/src/openstudio_app/OpenStudioApp.cpp

```diff
@@ -1134,16 +1102,16 @@ void OpenStudioApp::startMeasureManagerProcess(){
   }
 
   // find available port
-  QTcpServer* tcpServer = new QTcpServer(this);
-
-  tcpServer->listen(QHostAddress::LocalHost);
-  quint16 port = tcpServer->serverPort();
-  tcpServer->close();
-  delete tcpServer;
```

Comment:
I can see a race condition here where one process thinks it has a lock on a port but then another process finds the same port before this one is locked